R/rmarkdown.R

```diff
@@ -55,7 +55,7 @@ render_rmarkdown <- function(pkg, input, output, ..., copy_images = TRUE, quiet
   }
   if (digest != file_digest(output_path)) {
     href <- paste0("ide:run:pkgdown::preview_page('", path_rel(output_path, pkg$dst_path), "')")
-    cat_line("Writing ", cli::style_hyperlink(dst_path(output), href))
+    cli::cli_inform("Writing {cli::style_hyperlink(dst_path(output), href)}")
```

Comment:
Use `.href`?

Reply:
This is a complex href and best left as-is, because it works.